File: android/src/main/kotlin/dev/flutterbr/opentok_flutter/PublisherSettings.kt
package dev.flutterbr.opentok_flutter

import kotlinx.serialization.Serializable

@Serializable
data class PublisherSettings(
        var name: String?,
        var audioTrack: Boolean?,
        var videoTrack: Boolean?,
        var audioBitrate: Int?,
        var cameraResolution: String?,
        var cameraFrameRate: String?,
        var styleVideoScale: String?)

File: ios/Classes/PublisherSettings.swift
//
//  PublisherSettings.swift
//  opentok_flutter
//
//  Created by Marcelo Rangel on 17/08/20.
//

import Foundation

struct PublisherSettings: Codable {
    var name: String?
    var audioTrack: Bool?
    var videoTrack: Bool?
    var audioBitrate: Int?
    var cameraResolution: OTCameraCaptureResolution?
    var cameraFrameRate: OTCameraCaptureFrameRate?
    var styleVideoScale: String?
}
